src/content/getting-started/provision-your-first-workload-cluster/_index.md

```diff
@@ -236,88 +188,15 @@ error: the server doesn't have a resource type "organizations"
-
-In addition to workload clusters, you can also template applications. Apps belong to catalogs. While you can define your own catalogs, we already provide two: `giantswarm` (_Giant Swarm Catalog_) contains applications that we know how to manage; `giantswarm-playground` (_Playground_) contains applications that we've integrated but aren't managed by us. Other catalogs such as the _Giant Swarm Cluster Catalog_ are used for templating of clusters (which you did above).
-
-You can easily surf catalogs from the _Apps_ section in the Web UI:
-
-![App Catalog](apps.png)
-
-Let's assume you want to install the Kong app. By clicking on it, we can see its details:
-
-![The Kong application](kong-app.png)
-
-With the web interface, you can configure and deploy the app by clicking _Install in this cluster_. In this tutorial, we show how to template the app into a YAML manifest. The command can be seen at the bottom of the Web UI app overview, for instance:
-
-```sh
-kubectl gs template app \
-  --catalog giantswarm \
-  --name kong-app \
-  --target-namespace test-namespace \
-  --cluster-name o8r3r \
-  --version 3.3.0 \
-  > kong-app.yaml
-```
-
-This writes the `App` CR to the file `kong-app.yaml`.
-
-To deploy the app, deploy it, again using the management cluster context:
-
-```sh
-kubectl apply -f kong-app.yaml
-```
-
-Notice that by running `kubectl gs template app --help` you can list all the options that can be useful when templating an app. For instance, the `--user-configmap` and `--user-secret` options allow for adding YAML files containing useful configurations or secrets.
-
-Here is an example which disables the configuration flag `proxy.enabled` for Kong:
+## Step 4: Deleting the workload cluster {#deleting-workload-cluster}
 
-```sh
-cat > kong-config.yaml <<EOF
-proxy:
-  enabled: false
-EOF
-
-kubectl gs template app \
-  --catalog giantswarm \
-  --name kong-app \
-  --target-namespace test-namespace \
-  --cluster-name o8r3r \
-  --version 3.3.0 \
-  --user-configmap kong-config.yaml \
-  > kong-app.yaml
-```
-
-In general, whenever templating and/or installing an application, please read its documentation page on the Web UI to be sure you are configuring everything correctly.
-
-## Step 7: Updates of workload clusters and managed apps
-
-### Updating an application
+Deletion works in the same way: run `kubectl delete -f FILENAME.yaml` and the operators in the management cluster will delete the resources in a few minutes. Please don't directly delete the Cluster API custom resources (such as `Cluster`, `AWSCluster` or `MachineDeployment`) since this may leave resources behind or even lead to inadvertently recreating the cluster once the `App` is reconciled again. Deletion should be done exactly like the creation, using the original manifests. For the Cluster API product family, our example output file `cluster.yaml` contains 2 `App` and 2 `ConfigMap` manifests. If you no longer have the manifests at hand, delete the following:
```

Comment:
When starting to read this sentence, users may get confused about what `FILENAME.yaml` is supposed to be. We explain it later, so it's better to probably explain it first, then use `cluster.yaml` as file name.